.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto
```

Comment:
This ought to be fine, just wanted to share one other option for consideration:
```
*.js text
*.md text
etc for each file extension
```
Tradeoff is harder to maintain but less likely to mangle any binary files

Reply:
After reading more on this I see that explicit exclusion of paths as `text` is also supported and that leads me to support `* text=auto` rather than what I've mentioned above